test/remote/gateways/remote_digital_river_test.rb

```diff
@@ -0,0 +1,169 @@
+require 'test_helper'
+require 'digital_river'
+
+class RemoteDigitalRiverTest < Test::Unit::TestCase
+  def setup
+    @gateway = DigitalRiverGateway.new(fixtures(:digital_river))
+    @digital_river_backend = @gateway.instance_eval{@digital_river_gateway}
```

Comment:
I think that using the `instance_variable_get` method would be a better idea here:

```ruby
@gateway.instance_variable_get(:@digital_river_gateway)
```